### ISSUE TITLE
Investigate using slate-style schema for rich text

### ISSUE THREAD

**Issue by BogdanDogaru** [bbc/simorgh#525]
**Is your feature request related to a problem? Please describe.**

More background about this problem here: https://confluence.dev.bbc.co.uk/display/cps/Optimo+Paragraph+Blocks

**Describe the solution you'd like**

1. Investigate a solution to render the rich text in the 'slate-style' format isomorphically
2. Produce a proof-of-concept branch of Simorgh that would demonstrate how this would work
3. Identify issues or potential issues with regards to flexibility or client-side performance

**Describe alternatives you've considered**

At the moment we're using markdown, but for reasons mentioned in the link above, this solution is not flexible enough for our current and future needs.

**Additional context**

**Comment by jtart** [comment]
See the write-up in #587 for a summary on this investigation.

TLDR; a slate model is acceptable.
